pkg/utils/notify/notify.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package notify
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	securejoin "github.com/cyphar/filepath-securejoin"
+	"github.com/s3rj1k/go-fanotify/fanotify"
+	"golang.org/x/sys/unix"
+)
+
+// RuntimePaths is the list of paths where the container runtime runc or crun
+// could be installed. Depending on the Linux distribution, it could be in
+// different locations.
+//
+// When this package is executed in a container, it prepends the HOST_ROOT env
+// variable to the path.
+var RuntimePaths = []string{
+	"/bin/runc",
+	"/usr/bin/runc",
+	"/usr/sbin/runc",
+	"/usr/local/bin/runc",
+	"/usr/local/sbin/runc",
+	"/usr/lib/cri-o-runc/sbin/runc",
+	"/run/torcx/unpack/docker/bin/runc", // Used in TODO
+	"/usr/bin/crun",
+	"/usr/bin/conmon",
```

Comment:
"conmon" should be in the list for `container-hook` but not for `runcfanotify`, see https://github.com/inspektor-gadget/inspektor-gadget/pull/2764#discussion_r1583272674